app/models/movie.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Movie class
+class Movie < ApplicationRecord
+  has_and_belongs_to_many :people
+
+  validates :title, :release_year, presence: true
+
+  def people_as(role_name)
```

Comment:
please add comments. References to what does this method and other similars

Reply:
ready